e2e/extensions.e2e.spec.ts

```diff
@@ -201,5 +201,65 @@ test.describe.serial('Extensions', () => {
         hostname: os.hostname(),
       });
     });
+
+    test.describe('running host commands', () => {
+      const wrapperName = process.platform === 'win32' ? 'dummy.cmd' : 'dummy.sh';
+
+      test('capturing output', async() => {
+        const script = `
+          ddClient.extension.host.cli.exec("${ wrapperName }", [
+            "${ process.execPath }", "-e", "console.log(1 + 1)"
+          ]).then(({cmd, killed, signal, code, stdout, stderr}) => ({
+            /* Rebuild the object so it can be serialized properly */
+            cmd, killed, signal, code, stdout, stderr
+          }));
+        `.replace(/\s+/g, ' ');
```

Comment:
What's the reason for removing the whitespace? Does `executeJavaScript` stumble on newlines?